src/vivarium_gates_iv_iron/model_specifications/south_asia.yaml

```diff
@@ -31,7 +31,7 @@ configuration:
             day: 31
```

Comment:
We shouldn't need separate yaml files for each location. Simply specifying the artifact path in the `scenarios.yaml` file should be enough? You might also need to specify the location there as well.

Reply:
We'll have to consider this longer term. Multiple locations will be a big issue for at least one upcoming project.